packages/integrations/cloudflare/README.md

```diff
@@ -236,71 +269,16 @@ declare namespace App {
 }
 ```
 
-### Environment Variables
-
-See Cloudflare's documentation for [working with environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables).
-
-```js
-// pages/[id].json.js
-
-export function GET({ params }) {
-  // Access environment variables per request inside a function
-  const serverUrl = import.meta.env.SERVER_URL;
-  const result = await fetch(serverUrl + "/user/" + params.id);
-  return {
-    body: await result.text(),
-  };
-}
-```
-
-### `cloudflare.runtime`
-
-`runtime: "off" | "local" | "remote"`
-default `"off"`
-
-This optional flag enables the Astro dev server to populate environment variables and the Cloudflare Request Object, avoiding the need for Wrangler.
-
-- `local`: environment variables are available, but the request object is populated from a static placeholder value.
-- `remote`: environment variables and the live, fetched request object are available.
-- `off`: the Astro dev server will populate neither environment variables nor the request object. Use Wrangler to access Cloudflare bindings and environment variables.
-
-```js
-// astro.config.mjs
-import { defineConfig } from 'astro/config';
-import cloudflare from '@astrojs/cloudflare';
-
-export default defineConfig({
-  output: 'server',
-  adapter: cloudflare({
-    runtime: 'off' | 'local' | 'remote',
-  }),
-});
-```
-
-## Wasm module imports
-
-`wasmModuleImports: boolean`
-
-default: `false`
-
-Whether or not to import `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration).
+## Headers, Redirects and function invocation routes
```

Comment:
I know this wasn't you originally, but seeing attention called to it because of the edits... This sounds like... a lot! :sweat_smile:

(:lion: :tiger: :bear: ... oh my!)

So for bonus points, what really is going on here? If a heading is supposed to be a helpful signpost, it probably shouldn't be a long list. Can you abstract or generalize here? (also ok to leave as not your edit, so not your responsibility, and NWTWWHB!)

Reply:
I've restructured the section to be broken down. I've also added some generic content, but I might need some help with wording here :)